frameworks/PHP/fomo/config/server.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+return [
+    /*
+     * mode
+     * SWOOLE_BASE ,
+     * SWOOLE_PROCESS
+     */
+    'mode' => SWOOLE_BASE ,
+    'host' => '0.0.0.0',
+    'port' => 9000 ,
+    'sockType' => SWOOLE_SOCK_TCP ,
+    'additional' => [
+        'worker_num' => env('APP_WORKER_COUNT' , cpuCount() * 2) ,
+        /*
+         * log level
+         * SWOOLE_LOG_DEBUG (default)
+         * SWOOLE_LOG_TRACE
+         * SWOOLE_LOG_INFO
+         * SWOOLE_LOG_NOTICE
+         * SWOOLE_LOG_WARNING
+         * SWOOLE_LOG_ERROR
+         */
+        'log_level' => SWOOLE_LOG_DEBUG ,
+        'log_file' => storagePath('logs/fomo.log') ,
+    
+        /*
+        This key causes Fomo to receive a complete HTTP data packet and prevents it from receiving incomplete HTTP packets.
+        */
+        'open_http_protocol' => true,
+    ],
+
+    'ssl' => [
+        'ssl_cert_file' => null ,
+        'ssl_key_file' => null ,
+    ] ,
+
+    /*
+     * The following services are created for better performance in the program, only one object is created from them and they can be used throughout the program
+     */
+    'services' => [
+        Fomo\Services\Auth::class ,
+        Fomo\Services\Cache::class ,
+        Fomo\Services\Database::class ,
+        Fomo\Services\Elasticsearch::class ,
+        Fomo\Services\Language::class ,
+        Fomo\Services\Mail::class ,
+        Fomo\Services\Redis::class ,
+        Fomo\Services\Response::class ,
+        Fomo\Services\Validation::class ,
```

Comment:
Please remove unnecessary services: mail, redis, elasticsearch, ...

And the corresponding config files.